GitVersion.yml

```diff
@@ -1,8 +1,7 @@
 assembly-versioning-scheme: Major
-next-version: 3.0.0
-commit-message-incrementing: Disabled
```

Comment:
afaik this feature is enabled by default, meaning any commit within a PR will now automatically increase the version?

Reply:
This is about disabling some specific syntax that allows you to add syntax to your commit message to bump a version.

Other repos don't have this setting in the GitVersion.yml, so I see no reason to have it here.